manifests/overlays/additional/jupyter-amd-pytorch-notebook-imagestream.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/opendatahub-io/notebooks/blob/main/jupyter/amd/pytorch"
+    opendatahub.io/notebook-image-name: "AMD-PyTorch"
```

Comment:
> For Nvidia, CUDA is used [https://github.com/opendatahub-io/notebooks/blob/main/cuda/ubi9-python-3.9/Dockerfile#L7](https://www.google.com/url?q=https://www.google.com/url?q%3Dhttps://github.com/opendatahub-io/notebooks/blob/main/cuda/ubi9-python-3.9/Dockerfile%2523L7%26sa%3DD%26source%3Ddocs%26ust%3D1719915959856960%26usg%3DAOvVaw0jWSCpGuAq_habIknZUQ9c&source=chat&ust=1719998762598000&usg=AOvVaw3pUmlwfNoMW_hO3Vl8NVwP). I think the convention in the ecosystem is to use the technology name rather than the company name. So Nvidia -> CUDA, Intel -> Habana, AMD -> ROCm?

observation from @astefanutti, following discussion on the AMD support refinement document v2

Reply:
Intel is now Gaudi, they shelved the Habana name, btw https://docs.habana.ai/en/latest/Release_Notes/Release%20Notes%20v1.14.0.html

Reply:
@dibryant as it mentioned above could you please update this PR instead of using AMD to use ROCm?